doc/classes/EditorToaster.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorToaster" inherits="HBoxContainer" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Manages toast notifications within the editor.
+	</brief_description>
+	<description>
+		This object manages the functionality and display of toast notifications within the editor, ensuring timely and informative alerts are presented to users.
```

Comment:
You could link to the `get_editor_toaster()` method.

Reply:
I've got a reference underneath, do you think that is sufficient? Kind of based it off EditorSettings.xml.

![image](https://github.com/user-attachments/assets/01542706-be2e-46ee-842b-d4ea07af9dbb)

Reply:
I missed it 🙃

Reply:
All g